jinja2/base.html

```diff
@@ -17,6 +17,10 @@
         "noindex, nofollow"
     {%- endif -%}
   >
+
+  <!-- because this is non-blocking, preload as early as possible -->
+  {% include "includes/preload.html" %}
```

Comment:
Why is this included as a separate file rather than inline?

Reply:
I am a bit of a stickler for this kinda thing. I get a little startled ;) when I open an HTML file and it is this huge wall of code. This is especially true for `head` sections.

This allows me to have this isolated, it does not add any performance issue, and is simple to update in future.

With all of that said, I am not resistant to change so, if you would prefer I inline this, I am more than happy to do so.